docs/release-notes/4142-tensorboard-profiler-data-from-workers.txt

```diff
@@ -0,0 +1,16 @@
+:orphan:
+
+**New Features**
+
+-  TensorBoard:
+      -  For Keras trials, the TensorBoard profiling data from all workers is uploaded to the
```

Comment:
Be precise here.  A lower-case trial is a run of an experiment.  A "Keras trial" could be a TFKerasTrial or a Core API training that uses Keras.  So rephrase as:  `In TFKerasTrial, the TensorBoard profiling...`

Same for "PyTorch trials"